lmdeploy/vl/utils.py

```diff
@@ -15,6 +15,30 @@ def encode_image_base64(image: Image.Image) -> str:
     return base64.b64encode(buffered.getvalue()).decode('utf-8')
 
 
+def encode_file_base64(image_url: str) -> str:
```

Comment:
Can we still name it `encode_image_base64`?
The argument name can be `image`, which can be a URL or a file.